verl/utils/reward_score/duplex/duplex_r1.py

```diff
@@ -0,0 +1,680 @@
+# Copyright 2024 Bytedance Ltd. and/or its affiliates
```

Comment:
If it is coming from an open source project, please add the correct license to this file.